docs/sphinx/specification/cudaq/types.rst

```diff
@@ -4,30 +4,91 @@ The CUDA Quantum language extension provides certain fundamental types that are
 to quantum-classical computing. These types are provided via defined classical library
 types in C++ in the :code:`cudaq` namespace. 
 
-:code:`cudaq::qudit<Levels>`
-----------------------------
-**[1]** The :code:`cudaq::qudit` models a :math:`D`-level unit of quantum information. The state of
-this system (for :math:`N` qudits) can be described by a :math:`D`\ :sup:`N`\-dimensional vector in
-Hilbert space with the absolute square of all elements summing to 1. 
+Quantum dit (qudit)
```

Comment:
are you introducing this word, dit, here? I've never seen it used before. It may be better to be more explicit with 'Quantum Information of Arbitrary Dimension' or 'D-Level Quantum Systems and Information' or something similar. Then "we introduce the d-level unit of quantum information, or qudit ..." and use qudit from there on.

Reply:
`dit` is a "word" in the same way as `bit`, and `trit` are words. I have seen used on papers that talk about `qudit`s. However, this is, arguably, a wrong use of the word: it usually means "decimal digit" (with bit being "binary digit", and trit, "ternary digit").